index.html

```diff
@@ -869,17 +861,19 @@ <h2>DID Parameters</h2>
               </td>
               <td>
 A resource hash of the <a>DID document</a> to add integrity protection, as
-specified in [[?HASHLINK]]. The associated value MUST be an
-<a data-lt="ascii string">ASCII string</a>. <i>This parameter is non-normative.</i>
+specified in [[?HASHLINK]]. Support for this parameter is OPTIONAL. If
```

Comment:
Pinging @iherman to check his opinion on replacing the sentence _"This parameter is non-normative"_ with the sentence _"Support for this parameter is OPTIONAL"_, considering his earlier comment in https://github.com/w3c/did-core/issues/137#issuecomment-665155958.

Reply:
I think the two terms are not interchangeable. The parameter is non-normative (ie, its behavior cannot be considered as fully stable) and its usage is optional (whether it has a stable behavior or not). 

I guess the description should say "non-normative and OPTIONAL)"

Reply:
Requested change has been made.